cogeo_mosaic/utils.py

```diff
@@ -135,8 +135,8 @@ def _get_point(asset: str, coordinates: Tuple[float, float]) -> dict:
             )
 
             if not (
-                (src_dst.bounds[0] < lng_srs[0] < src_dst.bounds[2]) and # noqa
-                (src_dst.bounds[1] < lat_srs[0] < src_dst.bounds[3])
+                (src_dst.bounds[0] < lng_srs[0] < src_dst.bounds[2])
+                and (src_dst.bounds[1] < lat_srs[0] < src_dst.bounds[3])  # noqa
```

Comment:
I guess you might not need `# noqa` here, because of 
https://github.com/developmentseed/cogeo-mosaic/blob/60749264bd8375c482ca5420ccefd39964edb3ce/.pre-commit-config.yaml#L19-L22

Which my `flake8` wasn't picking up on